### PR TITLE
Antavo Destination documentation

### PR DESCRIPTION
**Pull request by bbantavo**
Proposed changes
Added Antavo Destination documentation files.

Merge timing
asap

Related issues (optional)

### CODE REVIEW

File: src/connections/destinations/catalog/antavo/index.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      - customer attributes be included in the Data section - make sure attribute names match your Antavo settings
+      
+   ![Enable Twilio Segment extension](images/1-antavo-select_type.png)
+   - Select the Loyalty event template to trigger loyalty events.
+      - in this template, you need to map the customer ID first, which should match with the Antavo settings
+      - if the multi-account extension is enabled in Antavo, you need to make sure to include the account ID
+      - event attributes can be included in the Data section - make sure attribute names match your Antavo settings
+
+   ![Enable Twilio Segment extension](images/2-antavo-map_fields.png)
+7. If you haven’t configured the Segment integration in Antavo, please go to the Modules menu and enable Twilio Segment Extension
```
[the remark this answers is not in the document]
Comment (bbantavo) [reply src/connections/destinations/catalog/antavo/index.md]: hi @stayseesong 

thanks for the suggestions, we have applied them, please check them

**Comment by tcgilbert** [comment]
@bbantavo @stayseesong - we can merge this as hidden once my requested changes are in. The destination is still pending review, so there is no destination id for this yet

**Comment by stayseesong** [comment]
@tcgilbert I don't see your requested changes

**Review by tcgilbert — CHANGES_REQUESTED** [review]

**Comment by tcgilbert** [comment]
> @tcgilbert I don't see your requested changes

@stayseesong sorry was left in pending! added my review now @bbantavo

File: src/connections/destinations/catalog/antavo/index.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+1. From your workspace's [Destination catalog page](https://app.segment.com/goto-my-workspace/destinations/catalog){:target="_blank"} search for **Antavo (Actions)**.
+2. Click **Add Destination**.
+3. Select an existing Source to connect to Antavo (Actions).
+4. Log in to Antavo and go to the **Settings > API Settings** and copy your Antavo **API key**.
+5. Paste the **API Key** in the destination settings in Segment.
+6. Configure your mappings to set events you want to sync to Antavo. You can choose from 2 actions: Send Loyalty Event and Send Profile Update.
+      - If the multi-account extension is enabled in Antavo, make sure to include the account ID.
+      - If customer attributes are included in the Data section - make sure attribute names match your Antavo settings.
+7. If you haven’t configured the Segment integration in Antavo, go to the **Modules** menu and enable the Twilio Segment Extension in Antavo.
```
[the remark this answers is not in the document]
Comment (bbantavo) [reply src/connections/destinations/catalog/antavo/index.md:19]: hi @tcgilbert 

we've applied the changes, please check

**Review by tcgilbert — APPROVED** [review]

**Comment by stayseesong** [comment]
@tcgilbert is this good to merge in even though there's no destination ID?

**Review by stayseesong — APPROVED** [review]

**Pull request merged**